cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -341,7 +371,7 @@ class HydratorPlusPlusTopPanelCtrl {
     this.previewRunning = true;
     let poll = this.dataSrc.poll({
       _cdapNsPath: '/previews/' + previewId + '/status',
-      interval: 5000
+      interval: 500
```

Comment:
Ok, I changed it to `1000` instead of the original `5000` so we actually stop the preview after ~1:00 minute instead of ~1:05 minute.